Data/Sentinel1_processing_guide.qmd

```diff
@@ -0,0 +1,62 @@
+
+## [Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) 
+[Sentinel-1 RTC](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-rtc) (Radiometric Terrain Correction) SAR Backscatter is a product processed from Sentinel-1 GRD data and compliant with [CEOS Analysis Ready Data for Land (CARD4L) specifications](https://ceos-dev.ceos.org/ard/) for Normalised Radar Backscatter (NRB) products. Orthorectification is based on Copernicus DEM and no speckle filtering is applied. ([Additional product information](https://sentinels.copernicus.eu/web/sentinel/sentinel-1-ard-normalised-radar-backscatter-nrb-product))
+
+## Copernicus Sentinel-1 Mosaics
+
+Sentinel-1 GRD monthly and yearly mosaics (available soon).
+
+It is planned that mosaics will be processed with the following Sentinel Hub processing options:
+
+- Orthorectification with Copernicus 10m/30m DEM
+- Radiometric Terrain Correction
+- No speckle filtering applied 
+
+## [Sentinel Hub processing options](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options) 
+
+Sentinel Hub offers the following processing options in the [Sentinel-1 GRD processing chain](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-chain):
+
+- [Backscatter coefficients](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options):
+
+    - beta0
+    - sigma0
+    - gamma0 (ellipsoid)
+    - gamma0 (terrain) &rarr; **gamma0 RTC can only be performed if orthorectification is enabled**
+
+- [Lee Speckle Filtering](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options) applied on source data after calibration and noise removal
+- [Orthorectification](https://documentation.dataspace.copernicus.eu/APIs/SentinelHub/Data/S1GRD.html#processing-options) with Range-Doppler terrain correction using one of the following DEMs:
+
+    - Copernicus 10m/30m DEM (10m resolution inside [39 European states](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model) including islands and 30m elsewhere.)
+    - Copernicus 30m DEM
+    - Copernicus 90m DEM 
+
+## [CloudFerro processing options](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html) 
+
+CloudFerro offers CARD-BS and COH6/COH12 products as a [service on demand](https://documentation.dataspace.copernicus.eu/APIs/On-Demand%20Production%20API.html): 
+
+- [Sentinel-1 (CARD-BS) BackScatter](https://documentation.dataspace.copernicus.eu/Data/Additional.html#sentinel-1-card-bs)
+
+    - This processing option contains gamma0 geometric terrain correction (orthorectification), which is identical to the gamma0 (ellipsoid) backscatter coefficient with orthorectification option in Sentinel Hub processing options. No Radiometric Terrain Correction (RTC) is applied to this product. 
```

Comment:
Please add a bullet list item "Additional information" that refers to https://creodias.eu/eodata/sentinel-1/sentinel-1-l2-backscatter-bs/